app/ide-desktop/lib/dashboard/src/layouts/InfoMenu.tsx

```diff
@@ -0,0 +1,81 @@
+/** @file A menu containing info about the app. */
+import * as React from 'react'
+
+import LogoIcon from 'enso-assets/enso_logo.svg'
+import * as common from 'enso-common'
+
+import * as modalProvider from '#/providers/ModalProvider'
+import * as textProvider from '#/providers/TextProvider'
+
+import * as aria from '#/components/aria'
+import MenuEntry from '#/components/MenuEntry'
+import Modal from '#/components/Modal'
+import FocusArea from '#/components/styled/FocusArea'
+
+import AboutModal from '#/modals/AboutModal'
+
+// ================
+// === InfoMenu ===
+// ================
+
+/** Props for an {@link InfoMenu}. */
+export interface InfoMenuProps {
+  readonly hidden?: boolean
+}
+
+/** A menu containing info about the app. */
+export default function InfoMenu(props: InfoMenuProps) {
+  const { hidden = false } = props
+  const { setModal } = modalProvider.useSetModal()
+  const { getText } = textProvider.useText()
+  const [initialized, setInitialized] = React.useState(false)
+
+  React.useEffect(() => {
+    requestAnimationFrame(() => {
```

Comment:
Small question: why do we do this in useEffect and inside the requestAnimationFrame ?

Reply:
good question... the rAF is unnecessary, removed. added a comment to clarify why it's in a useEffect (we need the initial classes so that the CSS transition works)

Reply:
Btw, just for the record - it's better to use `useLayoutEffect` + `rAF` here, because `useEffect` can't guarantee that the markup is present at the moment of calling the callback. Usually, this happens when we change state during render(this isn't happening here, but we need to keep this in mind). `useLayoutEffect` is always called __syncronously__ after render(once the markup is applied to the DOM).